### ISSUE TITLE
How do I include an Subpage in my footer?

### ISSUE THREAD

**Issue by L3Sch** [sruupl/batflat#138]
With {$page.content} I can display the Content of the page. So how do I select special Subpages with this method? 
Example:
I have an contact page in my footer.html an I want it as an Page to change via the page tool.

Can anyone help me please?

**Comment by brayozz** [comment]
As good as Batflat is, it's just as dead.

**Comment by L3Sch** [comment]
@brayozz thx, Do you have any idea to fix my Problem? And which flat cms is the best i.y.o.

**Comment by smitti** [comment]
I can´t help you, because I do not use Batflat anymore. It´s a nice CMS but it will not work with PHP 8. Regards Uwe

**Comment by L3Sch** [comment]
@smitti thanks for your comment, whats your alternative flat cms?

**Comment by smitti** [comment]
Well, evaluate Bludit CMS or Kirby CMS.

**Comment by brayozz** [comment]
> @brayozz thx, Do you have any idea to fix my Problem? And which flat cms is the best i.y.o.

I'm trying Bludit, but missing few fancy things that were in Batflat (i.e. file editor via admin panel)

**Comment by smitti** [comment]
> > @brayozz thx, Do you have any idea to fix my Problem? And which flat cms is the best i.y.o.
> 
> I'm trying Bludit, but missing few fancy things that were in Batflat (i.e. file editor via admin panel)

Yes, I know. Batflat is much better then Bludit, but Batflat is on its end of life.

**Comment by haydius** [comment]
> @smitti I can´t help you, because I do not use Batflat anymore. It´s a nice CMS but it will not work with PHP 8. Well, evaluate Bludit or Kirby. Regards Uwe

I'm working on a PHP8.1 rework of the CMS - Mine actually runs most functions without issues on the frontend but I've got a way to go on the backends still though this is on the backburner for a bit as I am quite busy on other works.
I'll add a responsee here when I am ready to drop a fork in.

> @L3Sch With {$page.content} I can display the Content of the page. So how do I select special Subpages with this method? Example: I have an contact page in my footer.html an I want it as an Page to change via the page tool.
> Can anyone help me please?

I have already done something similar with some custom widgets / embeds so will look to clean this up too for the above.
